Gemfile

```
@@ -85,7 +85,7 @@ gem 'valid_email', '>= 0.1.3', github: 'hallelujah/valid_email', ref: '486b860'
 gem 'view_component', '~> 3.0'
 gem 'webauthn', '~> 2.5.2'
 gem 'xmldsig', '~> 0.6'
-gem 'xmlenc', '~> 0.7', '>= 0.7.1'
+gem 'xmlenc', '0.8.0'
```

should we keep the version that allows patch changes? This would allow `0.8.1` automatically I think
```suggestion
gem 'xmlenc', '~> 0.8'
```

---

It would, but the gem is very rarely updated and I'm not confident in version updates being safe without some more explicit checks 😅

---

Coincidentally I'd tried that syntax earlier in #11590 (`gem 'rubocop', '~> 1.69'`) and it installed the outdated `.0` version (`1.69.0`) instead of the latest version (`1.69.1`) 🤷